### DISCUSSION TITLE
Does skywalking support spring cloud stream?

### DISCUSSION THREAD

**Discussion by magestacks** [apache/skywalking#8316] [Q&A]
Hi：

I have a question. Does skywalking support spring cloud starter stream rocketmq

Why do you ask this? Because when using spring cloud starter stream rocketmq, MQ producers and consumers are not in the same process

Using spring cloud starter stream rocketmq does not have this problem

Version number:
skywalking：8.6. 0
spring-cloud-starter-stream-rocketmq：2.2. 5.RELEASE

**Comment by wu-sheng** [comment]
I think this is out of SkyWalking's scope. We just accepted plugins from RocketMQ community. Please contact them to ask whether they have plans to enhance.

**Reply by magestacks** [reply]
OK, thank you very much for your answer！
